RealEstate/apps/core/templates/core/base.html

```diff
@@ -19,10 +20,38 @@
  </head>
 
  <body>
-   <a href={% url "admin:index" %}>Admin</a>
+   <nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="{% url 'home' %}"><span class="glyphicon glyphicon-home" aria-hidden="true"></span> Real Estate App</a>
+    </div>
+
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href={% url "admin:index" %}>Admin</a></li>
```

Comment:
Django only lets User instances with the 'is_staff' flag set to login into the Admin, so lets only show this link if user.is_staff is True.